includes/options/class-amp-options-manager.php

```diff
@@ -27,10 +27,10 @@ class AMP_Options_Manager {
 	 */
 	protected static $defaults = [
 		Option::THEME_SUPPORT           => AMP_Theme_Support::READER_MODE_SLUG,
-		Option::SUPPORTED_POST_TYPES    => [ 'post' ],
+		Option::SUPPORTED_POST_TYPES    => [ 'post' => true ],
 		Option::ANALYTICS               => [],
 		Option::ALL_TEMPLATES_SUPPORTED => true,
-		Option::SUPPORTED_TEMPLATES     => [ 'is_singular' ],
+		Option::SUPPORTED_TEMPLATES     => [ 'is_singular' => true ],
```

Comment:
I'm second-guessing why using an associative array is necessary. If an item is absent, then this is also treated as being unsupported. So a simple array of strings should be fine.

Reply:
Undone in 3a326c3